book/website/reproducible-research/vcs/vcs-git-merge.md

```diff
@@ -1,18 +1,19 @@
-# The `git merge` command
+(rr-vcs-merge)=
+# Merging Branches in Git
 
-Once you've finished up some work on a branch and you're are ready to integrate it to your main project (or any other branch) you can merge the branch that you worked on into the master branch or any other target branch of your interest.
-You can also use merging to combine work that other people have done with your own and vice versa.
+(rr-vcs-merge-command)=
+## The `git merge` Command
 
-To merge a branch, `branch_A`, into another branch, `branch_B`, switch to `branch_A` via:
+Once you have finished up some work on a branch and you are ready to integrate it to your main project (or any other branch), you can merge the branch that you worked on into the master branch or any other target branch of your interest.
+You can also use merging to combine work that other people have done with your own and vice versa.
 
-```bash
+To merge a branch, branch_A, into another branch, branch_B, switch to branch_A via:
+```
 git checkout branch_A
 ```
-
-
 Merge it into branch_B by:
 
-```bash
+```
```

Comment:
using `bash` and `python` in the code block allows markdown to style the code. though not strictly necessary for the book, it is not wrong to keep them in this file.